### DISCUSSION TITLE
How do I exclude pull requests that contain certain words?

### DISCUSSION THREAD

**Discussion by derekjtong** [community/community#134467] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

There's a bunch of PRs with a word in the title, let's call it "randomword". I want to filter out all all these PRs and only see PRs without "randomword" in the title. How do I do this?

I've tried `-randomword`, `-title:randomword`, etc. to no success. The first one actually only brings up PRs with "randomword" in it.

I've read the docs, chatgpt'd, and googled. But can't find the answer.

Is this filtering supported?

**Comment by Mahd25** [comment, marked as answer]
**Hello;**

`is:pr -in:title test`

The sign (-) before "in" causes the filtering to be reversed
In this example:
(Indicates PRs that do not have the word "test" in their title.)

![PRs-filtering](https://github.com/user-attachments/assets/06729bd2-29f7-44ca-b9a0-8e1ab31df9d6)
